wordpress-mobile/WordPress-iOS
# Fix post settings crash and ensure local image paths are always correct

Addendum to #12750

To test:

1. Create a new post in the app and save it as a draft.
2. Open the same post in the web editor.
3. Upload a featured image to the post and save it.
4. In the app, make sure the remote changes are synced (e.g. pull to refresh the post list) and then open the post again.
5. Open the post settings. The image should be loaded.

Additional info:

The featured image in pot settings never shows a local image (only an uploaded one). However, we decided to keep the code consistent and `featuredImageURLForDisplay` will always return the correct path.

Update release notes:
